liquidctl/driver/msi_7zw0led.py

```diff
@@ -0,0 +1,43 @@
+"""liquidctl drivers for MSI liquid coolers.
```

Comment:
If this module is meant to be the home of the drivers for many different MSI liquid coolers families (which makes sense), then I think it should be called `msi.py` or, at most, `msi_cooler.py`.